src/styles/card.css

```diff
@@ -0,0 +1,91 @@
+.body__card {
+  width: 200px;
+}
```

Comment:
You should remove this to the separate BEM block file according to BEM. It is an element, so it should be inside the block